srcup/cli.py

```diff
@@ -34,8 +36,30 @@ def single(
     api_key: str = typer.Option(..., envvar="WD_API_KEY", help="Watchdog API key"),
     owner_username: str = typer.Option('', help="Username of project owner. Ignored when --init is also present"),
     name: str = typer.Option('', help="Project name"),
-    comment: str = typer.Option('', help="Comment for the project")
+    comment: str = typer.Option('', help="Comment for the project"),
+    show_version: bool = typer.Option(False, '--version', '-v', help="Show the version of the app")
```

Comment:
There is a small section in the Typer docs about how to do  this the "right way". Binary flags are a bit weird in this tool in the sense that they show `--version` and `-no-version` when doing `--help`.

This also helps solve the 'DEFAULT_ARGUMENT' thing

Reply:
Aha, right. There should certainly be something to do this properly.

However, this is a good workaround that seems to be working very well for now. That's why I kept it that way.

Will look into their docs for more info.

Reply:
It shouldn't really be that hard from my quick scan of the docs. It's exactly what we want (versions flag implementation step-by-step)

Reply:
Updated the code to use callbacks and the is_eager flag for `--version` option